lib/carma/client/mule_soft_client.rb

```diff
@@ -15,7 +16,12 @@ class RecordParseError < StandardError; end
 
       def create_submission_v2(payload)
         with_monitoring do
-          res = do_post('v2/application/1010CG/submit', payload, config.settings.async_timeout)
+          res = if Flipper.enabled?(:cg1010_oauth_2_enabled)
```

Comment:
If flipper is on make submission request with auth token.

Reply:
I don't think we need to pass the user in here? The plan is to just turn it on/off for all users at the same time. Or am I missing something with how this works?